code/modules/atmospherics/gasmixtures/reaction_factors.dm

```diff
@@ -148,7 +149,8 @@
 		/datum/gas/bz = "BZ is consumed at 0.25 reaction rate",
 		/datum/gas/freon = "Freon is consumed at 2.75 reaction rate",
 		/datum/gas/healium = "Healium is produced at 3 reaction rate",
-		"Temperature" = "Can only occur between [HEALIUM_FORMATION_MIN_TEMP] - [HEALIUM_FORMATION_MAX_TEMP]. Higher temperature increases the reaction rate.",
+		"Pressure" = "The higher the pressure the faster the reaction rate goes.",
```

Comment:
See before? maybe? this is a little stilted, but I honestly don't really mind that much if you wanna keep it, I've got a weird sense for this stuff anyway

Reply:
I'll improve it later today when I have time.

Reply:
Changed.